sphinx.po

```diff
@@ -39,11 +39,9 @@ msgstr ""
 "{removed}"
 
 #: tools/templates/dummy.html:8
-#, fuzzy
 msgid "Deprecated since version {deprecated}, removed in version {removed}"
 msgstr ""
-"Obsolète depuis la version {deprecated}, sera retirée dans la version "
-"{removed}"
+"Obsolète depuis la version {deprecated}, retirée depuis la version {removed}"
```

Comment:
tu veux pas "retiré dans" ?

Reply:
C'est le « depuis » qui te gêne ? J'aime bien moi.
Et « supprimé depuis », « n'existe plus depuis » ?

Reply:
Voui, c'est le depuis. Boarf, comme tu veux.

Reply:
En fait il y a deux phrases distinctes depuis peu:

    Doc/tools/extensions/pyspecific.py:    _deprecated_label = 'Deprecated since version {deprecated}, will be removed in version {removed}'
    Doc/tools/extensions/pyspecific.py:    _removed_label = 'Deprecated since version {deprecated}, removed in version {removed}'

Reply:
Allez _supprimée dans_ et walouh.

Reply:
D'ailleurs je passe _supprimée_ au masculin.

Reply:
Ah @JulienPalard, nos messages se sont croisés.
Oui j'ai vu. Effectivement c'est mieux d'être cohérent entre les deux phrases.